### PR TITLE
cache roster doc fixes

### PR DESCRIPTION
**Pull request by The-Loeki**
### What does this PR do?
make salt ssh cache roster docs look great again

### CODE REVIEW

File: salt/roster/cache.py [hunk 1]
```diff
@@ -28,7 +28,6 @@
 overriding the one in ``roster_defaults``:
 
 .. code-block:: yaml
-
 roster_order:
```
Comment (rallytime) [inline salt/roster/cache.py]: I think for these to display as code blocks, the block of yaml lines needs to be indented beneath the code-block line? Like this:
```
.. code-block:: yaml
    roster_order:
      host: id
```

**Review by rallytime — COMMENTED** [review]
Thanks for this @The-Loeki! I had one comment about this change. I think for those docs to display the code-blocks, the blocks need to be indented.

File: salt/roster/cache.py (hunk 1 above)
Comment (The-Loeki) [inline salt/roster/cache.py]: oooooooooooooooo is that what's wrong w/it ;)

**Comment by The-Loeki** [comment]
Consider them indented rather than de-linespaced ;)

**Review by rallytime — APPROVED** [review]
Awesome!

**Pull request merged**